docs/ref/checks.txt

```diff
@@ -623,7 +623,8 @@ with the admin site:
 * **admin.E022**: The value of ``radio_fields`` refers to ``<field name>``,
   which is not an attribute of ``<model>``.
 * **admin.E023**: The value of ``radio_fields`` refers to ``<field name>``,
-  which is not a ``ForeignKey``, and does not have a ``choices`` definition.
+  which is not instance of ``ForeignKey``, and does not have a ``choices``
```

Comment:
"an" is missing in the correction: "not an instance"

Reply:
Thanks @timgraham for catching 🎯 . I will create a fix PR on Monday.